docs/community_resources/faq.md

```diff
@@ -0,0 +1,81 @@
+#FAQ
```

Comment:
Goes for all headers here, but you need to have a space between the hash and the text for it to be rendered as header.

Also makes it easier to read the markdown :)